CHANGELOG.md

```diff
@@ -66,9 +67,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - [1685](https://github.com/umee-network/umee/pull/1685) Add medians param to Token registry.
 - [1683](https://github.com/umee-network/umee/pull/1683) Add MaxBorrow query and allow returning all denoms from MaxWithdraw.
 - [1690](https://github.com/umee-network/umee/pull/1690) Add MaxBorrow message type.
-- [1711](https://github.com/umee-network/umee/pull/1711) Add historic pricing information to leverage MarketSummary query
-- [1715](https://github.com/umee-network/umee/pull/1715) Reverted.
-- [1723](https://github.com/umee-network/umee/pull/1723) Compute borrow limits using the lower of either spot or historic price for each collateral token, and the higher of said prices for borrowed tokens. Remove extra spot/historic only fields in account summary.
+- [1711](https://github.com/umee-network/umee/pull/1711) Add historic pricing information to leverage MarketSummary query.
+- [1723](https://github.com/umee-network/umee/pull/1723) Compute borrow limits using the lower of either spot or historic price for each collateral token, and the higher of said prices for borrowed tokens. Remove extra spot/historic only fields in account summum.
+- [1694](https://github.com/umee-network/umee/pull/1694) Add new sdkutil package to enhance common Cosmos SDK functionality. Here, the `ZeroCoin` helper function.
```

Comment:
```suggestion
```
Unobservable (from chain side) code refactoring probably doesn't need changelog unless it's breaking a library which other packages / repositories consume.

Reply:
But observable for dev side. Especially from other team who are using/copying our code, and potential other teams who may / will use it as a library.